website/docs/guides/visual-editor-qs.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Quickstart for the dbt Cloud Visual Editor"
+title: "Quickstart for the dbt Cloud Canvas"
 id: "visual-editor"
 level: 'Beginner'
 icon: 'dbt'
 hide_table_of_contents: true
-tags: ['Visual Editor','Analyst', 'dbt Cloud','model']
+tags: ['Canvas','Analyst', 'dbt Cloud','model']
```

Comment:
I think we should leave "visual editor" in here as well since it's a term some customers know. It will help with the search functionality if they use it
```suggestion
tags: ['Canvas','Analyst', 'dbt Cloud','model','visual editor']
```